### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by radishevskyi**
# 📝 Description

- added footer
- added adaptive
- fix styles
- added icons

# 🔄 Type of change

- [ ] Cleaning up
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality to something)
- [x] Configuration change
- [x] Translation change

# 🧪 How has this been tested / or will it be tested?

- [ ] Manually surfing the app

# ✅ Checklist:

- [x] My changes generate no new warnings in console or terminal
- [x] I have performed a self-review of my code
- [ ] I have updated versions of the package.json and package-lock.json

# 🖼️ Result screenshot/video (optional):
image

# Gif to summarize my emotions (optional):

### CODE REVIEW

File: src/shared/assets/icons/facebook.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { IconProps } from '@/shared/assets/icons/icons.types';
+
+const Facebook = ({ color, ...props }: IconProps) => (
+  <svg
+    data-icon="facebook"
+    width="44"
```
Comment (p-osypov) [inline src/shared/assets/icons/facebook.tsx:7]: All icons by default must be 20x20

File: src/shared/components/footer/footer.tsx [hunk 2]
```diff
@@ -1,8 +1,83 @@
...
+          <SC.About>
+            <SC.TitleLogo>
+              <SC.Logo src="/img/logo.png" />
+              <SC.Title>{t('name')}</SC.Title>
+            </SC.TitleLogo>
+            <SC.TextAboutDrone>{t('text')}</SC.TextAboutDrone>
+          </SC.About>
+          <SC.Contacts>
+            <SC.FooterTitle>{t('contacts')}</SC.FooterTitle>
+            <SC.FooterContactLink href="tel:{contacts.footerTel}">
```
Comment (p-osypov) [inline src/shared/components/footer/footer.tsx:30]:
```suggestion
<SC.FooterContactLink href={`tel:${contacts.footerTel}`}>
```

File: src/shared/components/footer/footer.tsx [hunk 3]
```diff
@@ -1,8 +1,83 @@
...
+              <Icons.FooterPhoneNum className="icon" />
+              {contacts.footerTel}
+            </SC.FooterContactLink>
+            <SC.FooterContactLink href="mailto:{contacts.footerEmail}">
```
Comment (p-osypov) [inline src/shared/components/footer/footer.tsx:34]: Same here

File: src/shared/components/footer/footer.tsx [hunk 4]
```diff
@@ -1,8 +1,83 @@
...
+                  {t(item.tranKey)}
+                </Link>
+              ))}
+            </SC.InfoList>
+          </SC.Info>
+          <SC.SocialNetworks>
+            <SC.FooterTitle>{t('socialNetworks')}</SC.FooterTitle>
+            <SC.FooterIcons>
+              <SC.SocialLink
+                href={'https://www.facebook.com/chornyi.stryzh/?_rdr'}
```
Comment (p-osypov) [inline src/shared/components/footer/footer.tsx:53]: Move social network url to contacts object. Remove redundant values like `/?_rdr`

File: src/shared/components/footer/footer.tsx [hunk 5]
```diff
@@ -1,8 +1,83 @@
...
+                  'https://www.youtube.com/@%D0%A0%D0%A3%D0%91%D0%BF%D0%90%D0%9A%D0%A7%D0%BE%D1%80%D0%BD%D0%B8%D0%B9%D0%A1%D1%82%D1%80%D0%B8%D0%B6'
+                }
+              >
+                <Icons.Youtube />
+              </SC.SocialLink>
+            </SC.FooterIcons>
+          </SC.SocialNetworks>
+        </SC.Content>
+        <SC.FooterBottom>
+          <span>© Yoursitename 2023 | All Rights Reserved</span>
```
Comment (p-osypov) [inline src/shared/components/footer/footer.tsx:71]:
```suggestion
<span>`© ${t('name)} ${currentYear} | ${t('allRights')}`</span>
```

File: src/shared/components/footer/footer.tsx [hunk 6]
```diff
@@ -1,8 +1,83 @@
...
+          <SC.FooterBottomLinks>
+            <Link href={'/'}>Trams & Condition</Link>
```
Comment (p-osypov) [inline src/shared/components/footer/footer.tsx:73]: Translations missed

**Review by p-osypov — CHANGES_REQUESTED** [review]
Don't forget to test media after these updates.

File: src/shared/components/footer/footer.tsx (hunk 3 above)
Comment (radishevskyi) [reply src/shared/components/footer/footer.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/4/commits/0d338980483aa1d61d8bc76b8ce4afd5854ee743)

File: src/shared/components/footer/footer.tsx (hunk 2 above)
Comment (radishevskyi) [reply src/shared/components/footer/footer.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/4/commits/0d338980483aa1d61d8bc76b8ce4afd5854ee743)

File: src/shared/components/footer/footer.tsx (hunk 5 above)
Comment (radishevskyi) [reply src/shared/components/footer/footer.tsx]: [fix: logic](https://github.com/p-osypov/army/pull/4/commits/0d338980483aa1d61d8bc76b8ce4afd5854ee743)

File: src/shared/components/footer/footer.tsx (hunk 4 above)
Comment (radishevskyi) [reply src/shared/components/footer/footer.tsx]: [feat: added social network links](https://github.com/p-osypov/army/pull/4/commits/db69025222b27728239455956005a9b55b52c898)
[fix: logic](https://github.com/p-osypov/army/pull/4/commits/0d338980483aa1d61d8bc76b8ce4afd5854ee743)

File: src/shared/components/footer/footer.tsx (hunk 6 above)
Comment (radishevskyi) [reply src/shared/components/footer/footer.tsx]: [feat: added translation](https://github.com/p-osypov/army/pull/4/commits/60bb5d30b101f23b6227f9f614f95fde71afe0d9)

File: src/shared/assets/icons/facebook.tsx (hunk 1 above)
Comment (radishevskyi) [reply src/shared/assets/icons/facebook.tsx]: [fix: icon sizes](https://github.com/p-osypov/army/pull/4/commits/5f812b789125c3920ca4ebdecdc02faf38104c8a)

**Review by p-osypov — APPROVED** [review]

**Pull request merged**